Dockerfile

```
@@ -132,7 +132,8 @@ RUN echo "source ~/ros2_ws/install/setup.bash" >> ~/.bashrc && \
     echo "alias cm='cd ~/ros2_ws;colcon build;source ~/.bashrc'" >> ~/.bashrc && \
     echo "alias cs='cd ~/ros2_ws/src'" >> ~/.bashrc && \
     echo "alias cw='cd ~/ros2_ws'" >> ~/.bashrc && \
-    echo "alias waypoint_manager='python3 /home/ubuntu/ros2_ws/src/orange_navigation/waypoint_manager/manager_GUI.py'" >> ~/.bashrc
+    echo "alias waypoint_manager='python3 /home/ubuntu/ros2_ws/src/orange_navigation/waypoint_manager/manager_GUI.py'" >> ~/.bashrc && \
+    echo "alias map_merger='python3 /home/ubuntu/ros2_ws/src/multi_map_manager/map_merger.py'" >> ~/.bashrc
```

こちらのPRは動作確認していますか？
以下のpathに間違いがあると思います。
```suggestion
    echo "alias map_merger='python3 /home/ubuntu/ros2_ws/src/multi_map_manager/map_merger/map_merger.py'" >> ~/.bashrc
```

---

pathの修正, map_trimme.pyのalias追加完了しました. ご確認お願いいたします.